app/api/chat/route.ts

```diff
@@ -23,9 +24,26 @@ export async function POST(req: Request) {
       });
     }
 
+    // Build system prompt with artist knowledge if available
+    let systemPrompt = DESCRIPTION;
+    
+    if (room_id) {
+      const artistKnowledge = await getKnowledgeBaseContext(room_id);
+      
+      if (artistKnowledge) {
+        systemPrompt = `${DESCRIPTION}
+
+-----ARTIST KNOWLEDGE BASE-----
+${artistKnowledge}
+-----END KNOWLEDGE BASE-----
+
+When answering questions, incorporate relevant information from the artist knowledge base above.`;
+      }
+    }
```

Comment:
could you please create a new lib called `getSystemPrompt`?

Let's add any context needed by the MCP tools into that lib
- knowledge base
- artist_account_id
- etc